### DISCUSSION TITLE
Add Quicksort

### DISCUSSION THREAD

**Discussion by ayyash1** [avelino/awesome-go#5441] [Ideas]
I would like to propose adding a Quick Sort algorithm implementation to the repository. Quick Sort is a widely used sorting algorithm that offers efficient performance, especially for large datasets. Including this feature would enhance the utility of the repository and provide users with an additional sorting option.

**Comment by phanirithvij** [comment]
This isn't really a repository for algorithms.
see https://github.com/TheAlgorithms/Go
and other related lists, https://github.com/williamfiset/Algorithms, https://github.com/tayllan/awesome-algorithms
